docs/pages/developing-in-workspaces/create-a-workspace.mdx

```diff
@@ -33,6 +33,16 @@ Then press `Create Workspace` to launch the workspace.
 Under the hood, the Desktop Application will call the CLI command `devpod up REPOSITORY`
 :::
 
+:::info Note
+You can set the location of your devpod home by passing the `--devpod-home={home_path}` flag, 
+or by setting the env var `DEVPOD_HOME` to your desired home directory.
+
+This can be useful if you are having trouble with a workspace trying to mount to a windows location when it should be mounting
+to a path inside the WSL VM.
+
+For example: setting `devpod-home=/mnt/c/Users/MyUser/` will result in a workspace mount path of something like `/mnt/c/Users/MyUser/.devpod/agent/contexts/default/workspaces/...`
```

Comment:
the `agent` directory only appears here if you're using a local provider, can you remove that pls? This might confuse people if they aren't able to find it on their machine because they are using a remote provider